content/en/serverless/guide/gcr_serverless_init.md

```diff
@@ -0,0 +1,194 @@
+
+Images are tagged based on semantic versioning, with each new version receiving three relevant tags:
+
+* `1`, `1-alpine`: use these to track the latest minor releases, without breaking changes
+* `1.x.x`, `1.x.x-alpine`: use these to pin to a precise version of the library
+* `latest`, `latest-alpine`: use these to follow the latest version release, which may include breaking changes
+
+## How `serverless-init` works
+
+The `serverless-init` application wraps your process and executes it as a subprocess. It starts a DogStatsD listener for metrics and a Trace Agent listener for traces. It collects logs by wrapping the stdout/stderr streams of your application. After bootstrapping, serverless-init then launches your command as a subprocess.
+
+To get full instrumentation, ensure you are calling `datadog-init` as the first command that runs inside your Docker container. You can do this through by setting it as the entrypoint, or by setting it as the first argument in CMD.
+
+{{< programming-lang-wrapper langs="nodejs,python,java,go,dotnet,ruby,php" >}}
+{{< programming-lang lang="nodejs" >}}
+
+{{% svl-init-nodejs %}}
+
+{{< /programming-lang >}}
+{{< programming-lang lang="python" >}}
+
+{{% svl-init-python %}}
+
+{{< /programming-lang >}}
+{{< programming-lang lang="java" >}}
+
+{{% svl-init-java %}}
+
+{{< /programming-lang >}}
+{{< programming-lang lang="go" >}}
+
+{{% svl-init-go %}}
+
+{{< /programming-lang >}}
+{{< programming-lang lang="dotnet" >}}
+
+{{% svl-init-dotnet %}}
+
+{{< /programming-lang >}}
+{{< programming-lang lang="ruby" >}}
+
+{{% svl-init-ruby %}}
+
+{{< /programming-lang >}}
+{{< programming-lang lang="php" >}}
+
+{{% svl-init-php %}}
+
+{{< /programming-lang >}}
+{{< /programming-lang-wrapper >}}
+
+### Buildpack
+
+[`Pack Buildpacks`][3] provide a convenient way to package your container without using a Dockerfile.
+
+First, manually install your tracer:
+- [Node.JS][14]
+- [Python][13]
+- [Java][15]
+- [Go][12]
+- [.NET][18]
+- [Ruby][16]
+- [PHP][17]
+
+Then, build your application by running the following command:
+
+```shell
+pack build --builder=gcr.io/buildpacks/builder \
+--buildpack from=builder \
+--buildpack datadog/serverless-buildpack:latest \
+gcr.io/<YOUR_PROJECT>/<YOUR_APP_NAME>
+```
+
+**Note**: Buildpack instrumentation is not compatible with Alpine images.
+
+## Configure your application
+
+Once the container is built and pushed to your registry, the last step is to set the required environment variables for the Datadog Agent:
+- `DD_API_KEY`: Datadog API key, used to send data to your Datadog account. It should be configured as a [Google Cloud Secret][11] for privacy and safety.
+- `DD_SITE`: Datadog endpoint and website. Select your site on the right side of this page. Your site is: {{< region-param key="dd_site" code="true" >}}.
+- `DD_TRACE_ENABLED`: Set to `true` to enable tracing.
+- `DD_TRACE_PROPAGATION_STYLE`: Set this to `datadog` to use context propagation and log trace correlation.
```

Comment:
Both of these are set by default. I would remove them from the doc to simplify the setup.